include/deal.II/matrix_free/tensor_product_kernels.h

```diff
@@ -2982,6 +2982,56 @@ namespace internal
 
 
 
```

Comment:
Could you add some documentation? The meaning of the function name isn't obvious to me.

Reply:
I have added a small comment. Do you think this is enough?